build/common.props

```diff
@@ -16,8 +16,8 @@
     <PackageReference Include="Internal.AspNetCore.Sdk" Version="$(InternalAspNetCoreSdkVersion)" PrivateAssets="All" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework' AND '$(OutputType)'=='library'">
-    <PackageReference Include="NETStandard.Library" Version="$(BundledNETStandardPackageVersion)" />
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework'">
```

Comment:
Does this sound better:

1. For `src` projects that target `.NETFramework`, add a reference to `NS.Library`. This keeps compat with 1.x
2. For samples and tests, we add a reference to `NS.Library.NETFramework`. We might require that they don't specifyy `PrivateAssets` to support publishing tests (plus it really doesn't matter for them).